modules/navier_stokes/test/tests/finite_volume/ins/natural_convection/linear_segregated/2d/tests

```diff
@@ -10,4 +10,13 @@
     max_threads = 1
     requirement = 'The system shall be able to use the density Boussinesq approximation to solve for a differentially heated 2D cavity.'
   []
+  [diff_heated_cavity_linear_buoyancy]
```

Comment:
Could you also add the test for triangles? Just to make sure we don't break it accidentally.
It could be just a modification using the `cli_args` parameter in the test.

Reply:
Done